### PR TITLE
Search without private group

### PR DESCRIPTION
**Pull request by rldhont**
Search controller and FTS does not use appContext

* Funded by 3liz

### CODE REVIEW

**Comment by Gustry** [comment]
FTS ? Is-it not linked to only QuickFinder ? Or is-it something else ?

**Comment by rldhont** [comment]
Cypress green

**Review by mdouchin — APPROVED** [review]
LGTM

**Comment by rldhont** [comment]
Rebase to get cypress updated to 9.5.0 and Cypress tests are still green.

File: lizmap/modules/lizmap/classes/lizmapFts.class.php [hunk 1]
```diff
@@ -11,9 +11,7 @@ class lizmapFts
 {
     protected $sql;
 
-    protected $project;
-
-    protected function setSql()
+    protected function setSql($project)
```
Comment (laurentj) [inline lizmap/modules/lizmap/classes/lizmapFts.class.php:14]: I recommend to rename this method to something like `generateSql`, as it is not really a setter method (i.e, the parameter is not a value to set).

**Review by laurentj — APPROVED** [review]
Ok for me, if the method setSql is renamed.

File: lizmap/modules/lizmap/classes/lizmapFts.class.php (hunk 1 above)
Comment (rldhont) [reply lizmap/modules/lizmap/classes/lizmapFts.class.php]: Method renamed

**Comment by rldhont** [comment]
Thanks @laurentj

**Review by laurentj — APPROVED** [review]

**Pull request merged**